x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_help_popover.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { HelpPopover, HelpPopoverButton } from '../components/help_popover/help_popover';
+
+export const TimeSeriesExplorerHelpPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  return (
+    <HelpPopover
+      anchorPosition="upCenter"
+      button={
+        <HelpPopoverButton
+          onClick={() => {
+            setIsPopoverOpen(!isPopoverOpen);
+          }}
+        />
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      isOpen={isPopoverOpen}
+      title={i18n.translate('xpack.ml.timeSeriesExplorer.popoverTitle', {
+        defaultMessage: 'Single time series analysis',
+      })}
+    >
+      <p>
+        <FormattedMessage
+          id="xpack.ml.timeSeriesExplorer.popoverBasicExplanation"
+          defaultMessage="Anomaly detection jobs analyze your data and create models to identify anomalous patterns of behavior. If the job has model plot enabled, you can view a chart that represents the actual and expected values over time."
```

Comment:
> If the job has model plot enabled, you can view a chart that represents the actual and expected values over time.

In https://github.com/elastic/kibana/blob/c13ae7ea6430f0bee954250f8e8ecb7c9f3e81a3/x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/date_histogram.tsx#L423 the specific setting is pulled from the code. Is this feasible with the model_plot_config setting here too?